tools/bumper/parser.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"reflect"
+
+	"github.com/pkg/errors"
+
+	"gopkg.in/yaml.v2"
+)
+
+type metadata struct {
+	Tag		string `yaml:"tag"`
+}
+
+type component struct {
+	Url         string `yaml:"url"`
+	Commit      string `yaml:"commit"`
+	Metadata 	metadata
+}
+
+type ComponentsList struct {
```

Comment:
Call this `Components` or `ComponentList`

Reply:
Done